HRCounter/HRCounter.csproj

```diff
@@ -42,10 +42,10 @@
     <DisableZipRelease>True</DisableZipRelease>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="0Harmony, Version=2.0.2.0, Culture=neutral, PublicKeyToken=null">
+    <Reference Include="0Harmony, Version=2.7.0.0, Culture=neutral, PublicKeyToken=null">
       <HintPath>$(BeatSaberDir)\Libs\0Harmony.dll</HintPath>
     </Reference>
-    <Reference Include="BSML, Version=1.5.2.0, Culture=neutral, processorArchitecture=MSIL">
+    <Reference Include="BSML, Version=1.6.1.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Please revert the changes to the version of the dependencies since I want this mod to work on the old versions of games.

Reply:
Reverted BSML changes in csproj